src/tl/result.rs

```diff
@@ -65,6 +66,9 @@ pub enum TonResult {
     #[serde(rename = "blocks.header")]
     BlocksHeader(BlocksHeader),
     // tonlib_api.tl, line 228
+    #[serde(rename = "configInfo")]
+    ChainConfigInfo(ChainConfigInfo),
```

Comment:
I would suggest preserving the original naming from tonlob_api.tl: ConfigInfo

Reply:
However this one, I was going to do the same in the first place. My problem was that a `ConfigInfo` is [already defined ](https://github.com/ston-fi/tonlib-rs/blob/main/src/tl/types.rs#L41)for `options.configInfo` in TL types. Just tell me how we do it without conflicts and I'll do it.

Reply:
Resolved!